components/tag-list/tag-list.js

```diff
@@ -140,7 +142,7 @@ class TagList extends LocalizeCoreElement(ArrowKeysMixin(LitElement)) {
 	}
 
 	focus() {
-		if (this._items.length > 0) this._items[0].focus();
+		if (this._items && this._items.length > 0) this._items[0].focus();
 	}
 
 	_chomp() {
```

Comment:
I wish GitHub would let me comment on lines outside your changes.  😦 

Is the purpose of this button simply for measurement?  If so, does the measurement ever change based on the number of items being chomped, or does it simply assume a width with the `##` value?  If it never changes, then perhaps we can avoid measuring every time we chomp.  It does seem like assuming a fixed with button is much easier to handle.